### PR TITLE
Features/consumer purchases and order details screen

### PR DESCRIPTION
**Pull request by OngJiaYuan**
Thank you for your contribution to the repo. 
Before submitting this PR, please make sure:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved terminology
- [x] You have added unit tests

You may delete the sections as necessary.

### Context
___

Fill this up with some context that will be useful to the reviewer that is looking at your PR.

image

This will be the starter screen when accessing the second tab from bottom navigation bar

image

upon clicking the bag icon
(Note picture used is surrounded by a black container to show how much space the picture can hold and this will be colorless in real time)

image

it will lead to this screen and upon clicking checkout below

image

it will lead to this screen 

image

Upon clicking on the credit/debit card it will lead to this screen 

image

This is the end of the first screen committed for the credit card screen 

The other commit which includes a list view builder as seen in the first picture upon clicking any of the item

image

it will lead to this screen 

image

which has two expandable widget 

image

This commit will be coupled with Man Hen commit so that upon clicking on checkout it will lead to his screen and his credit card screen when clicking on add card it will lead to the credit card screen
### Changes
___

Fill this up with the changes that are included in this PR.

Example:
Added test for all screens shared and passed all test 
image

### Shoutouts
___
Thank you Jospeh and Terence for the help

### CODE REVIEW

**Comment by OngJiaYuan** [comment]
[[Consumer] Purchases & Order detail](https://trello.com/c/xOcylbF0/17-consumer-purchases-order-detail)

**Comment by OngJiaYuan** [comment]
[[Consumer] Checkout (Add new payment)](https://trello.com/c/p16Mrpb3/16-consumer-checkout-add-new-payment)

File: lib/screens/brand_order/brand_order_list.dart [hunk 1]
```diff
@@ -52,9 +54,9 @@ class BrandOrderListState extends State<BrandOrderList> {
 
   String returnsItems(int num1) {
     if (num1 == 1) {
-      return '1 item';
+      return LocaleText.textItemsOne.tr();
     } else {
-      return num1.toString() + ' items';
+      return num1.toString() + ' ' + LocaleText.textItems.tr();
```
Comment (wheel1992) [inline lib/screens/brand_order/brand_order_list.dart:59]: You can do the follow:

```js
// en-US.json

"items": {
  "zero": "Item",
  "one": "Item",
  "two": "Items",
  "few": "Items",
  "many": "Items",
  "other": "Items"
}

// widget.dart
...
LocaleText.textItems.plural(num1)
...




```

File: lib/screens/consumer_order_credit_card/consumer_order_credit_card_screen.dart [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+  @override
+  _ConsumerOrderAddPaymentScreenState createState() =>
+      _ConsumerOrderAddPaymentScreenState();
+}
+
+class _ConsumerOrderAddPaymentScreenState
+    extends State<ConsumerOrderAddPaymentScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```
Comment (wheel1992) [inline lib/screens/consumer_order_credit_card/consumer_order_credit_card_screen.dart:16]: Use `VueScaffold` as it darken the Android status bar

File: lib/screens/consumer_purchases_and_order_details/consumer_purchases_and_order_details_screen.dart [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+  @override
+  _ConsumerPurchasesAndOrderDetailsScreenState createState() =>
+      _ConsumerPurchasesAndOrderDetailsScreenState();
+}
+
+class _ConsumerPurchasesAndOrderDetailsScreenState
+    extends State<ConsumerPurchasesAndOrderDetailsScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
```
Comment (wheel1992) [inline lib/screens/consumer_purchases_and_order_details/consumer_purchases_and_order_details_screen.dart:20]: Use `VueScaffold` here too

File: lib/screens/brand_order/brand_order_list.dart (hunk 1 above)
Comment (OngJiaYuan) [reply lib/screens/brand_order/brand_order_list.dart:59]: so for the plural when you input a number is it like a index kind or how does it work

**Pull request merged**